MPChartLib/src/main/java/com/github/mikephil/charting/charts/BarLineChartBase.java

```diff
@@ -1671,4 +1680,66 @@ protected void onSizeChanged(int w, int h, int oldw, int oldh) {
             mViewPortHandler.refresh(mViewPortHandler.getMatrixTouch(), this, true);
         }
     }
+    /**
+     * intercept single tapped event of chart
+     *
+     */
+    @Override
+    public void onChartSingleTapped(MotionEvent me) {
+        // Touched chart entry
+        Entry entry = getEntryByTouchPoint(me.getX(), me.getY());
+        if (entry != null) {
+            selectedYIndex = me.getY();
+        } else {
+            selectedYIndex = null;
+        }
+    }
+
+    @Override
+    public void onChartGestureStart(MotionEvent me, ChartTouchListener.ChartGesture lastPerformedGesture) {
+        Log.i("Gesture", "START");
```

Comment:
is it need logs?
is it tested graph?

Reply:
Check line chart and bar chart also

Reply:
Logs are not required, so i removed it. These changes are tested. i removed this handling for line chart.